app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id         :integer          not null, primary key
+#  name       :string
+#  email      :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class User < ApplicationRecord
+  has_many :reviews ,dependent: :destroy
+  has_many :restaurants, :through => :reviews
```

Comment:
Should use new-hash syntax throughout the code

https://github.com/WeConnect/style-guide/tree/master/ruby#syntax

Reply:
did you mean
has_many :reviews ,dependent: :destroy, :restaurants, :through => :reviews
?